### ISSUE TITLE
EditorNode3DGizmo::add_unscaled_billboard improper scaling

### ISSUE THREAD

**Issue by IlyaStuurFedorov** [godotengine/godot#52606]
### Godot version

3.3.3.stable

### System information

Windows 10

### Issue description

Currently, in the implementation of `add_unscaled_billboard` there seems to be an unnecessary multiplication by 100
```
mesh->set_custom_aabb(AABB(Vector3(-selectable_icon_size, -selectable_icon_size, -selectable_icon_size) 
* 100.0f, Vector3(selectable_icon_size, selectable_icon_size, selectable_icon_size) * 200.0f));
```

This makes `p_scale` higher than `0.05` overkill (this scale is used everywhere else in the editor for 3d billboard gizmos, which is why this problem might have been overlooked) and `nothing` around gizmo becomes clickable at this point.

### Steps to reproduce

* Creating a basic `EditorSpatialGizmoPlugin`, that calls `add_unscaled_billboard` on `redraw` with a scale larger than `0.05`, even default argument of `1.0` would showcase this issue.
* Realize that icon itself is pretty small, while the clickable area around it grows exponentially, making the huge area around gizmo unclickable.

**Comment by Calinou** [comment]
If this is exposed to scripting, this can only be fixed in 4.0 to avoid breaking existing plugins. In `3.x`, we can add a note in the documentation to give a suggested `p_scale` value range.

**Comment by IlyaStuurFedorov** [comment]
It seems that I made few mistakes while setting up material for billboard use, after looking into the source code more in-depth I found out how the material is being set up by the editor itself, and now it works exactly like every other billboard. Code had to look something like this:
```
godot::Ref<godot::SpatialMaterial> new_material;
new_material.instance();

godot::Ref<godot::Texture> texture = godot::ResourceLoader::get_singleton()->load("res://Textures/Editor/Script.png");

new_material->set_flag(godot::SpatialMaterial::Flags::FLAG_UNSHADED, true);
new_material->set_flag(godot::SpatialMaterial::FLAG_ALBEDO_FROM_VERTEX_COLOR, true);
new_material->set_flag(godot::SpatialMaterial::FLAG_SRGB_VERTEX_COLOR, true);
new_material->set_cull_mode(godot::SpatialMaterial::CULL_DISABLED);
new_material->set_depth_draw_mode(godot::SpatialMaterial::DEPTH_DRAW_DISABLED);
new_material->set_feature(godot::SpatialMaterial::Feature::FEATURE_TRANSPARENT, true);
new_material->set_texture(godot::SpatialMaterial::TextureParam::TEXTURE_ALBEDO, texture);
new_material->set_flag(godot::SpatialMaterial::FLAG_FIXED_SIZE, true);
new_material->set_billboard_mode(godot::SpatialMaterial::BillboardMode::BILLBOARD_ENABLED);
new_material->set_render_priority(godot::SpatialMaterial::RENDER_PRIORITY_MIN);

add_material("main", new_material);
```
